Getting errors trying to run example

I'm following the README instructions for the example. I keep getting an error:

```
> $ dgit -t cow-fortune fortune:latest cowsay:latest                            
Error: unknown command "fortune:latest" for "dgit"
Run 'dgit --help' for usage.
```

Tried using the `merge` command but the didn't help. running `test.sh` throws errors:

```
> $ ./test.sh                                                                       [±main ✓]
Running go tests...
go: downloading github.com/sirupsen/logrus v1.6.0
go: downloading github.com/docker/docker v17.12.0-ce-rc1.0.20180131172010-e1f98e8ab3cb+incompatible
go: downloading github.com/google/go-containerregistry v0.4.0
go: downloading github.com/containers/image v0.0.0-20180222154204-f7ea1dcb32a0
go: downloading github.com/spf13/cobra v1.0.0
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/google/go-github v15.0.0+incompatible
go: downloading golang.org/x/sys v0.0.0-20200930185726-fdedc70b468f
go: downloading github.com/spf13/pflag v1.0.5
go: downloading github.com/google/go-querystring v0.0.0-20170111101155-53e6ce116135
go: downloading github.com/opencontainers/image-spec v1.0.1
go: downloading github.com/opencontainers/go-digest v1.0.0
go: downloading github.com/docker/go-connections v0.4.0
go: downloading golang.org/x/net v0.0.0-20201110031124-69a78807bb2b
go: downloading github.com/docker/libtrust v0.0.0-20160708172513-aabc10ec26b7
go: downloading github.com/containerd/stargz-snapshotter/estargz v0.0.0-20201223015020-a9a0c2d64694
go: downloading golang.org/x/sync v0.0.0-20201207232520-09787c993a3a
go: downloading github.com/docker/distribution v2.7.1+incompatible
go: downloading github.com/docker/go-units v0.4.0
go: downloading github.com/gogo/protobuf v1.3.1
pkg/dgit/generate/generate.go:14:2: missing go.sum entry for module providing package github.com/davecgh/go-spew/spew (imported by matt-rickard.com/docker-merge/pkg/dgit/generate); to add:
        go get matt-rickard.com/docker-merge/pkg/dgit/generate
pkg/dgit/generate/generate.go:15:2: no required module provides package github.com/google/go-containerregistry/authn; to add it:
        go get github.com/google/go-containerregistry/authn
pkg/dgit/generate/generate.go:16:2: no required module provides package github.com/google/go-containerregistry/name; to add it:
        go get github.com/google/go-containerregistry/name
pkg/dgit/generate/generate.go:17:2: no required module provides package github.com/google/go-containerregistry/v1; to add it:
        go get github.com/google/go-containerregistry/v1
pkg/dgit/generate/generate.go:18:2: no required module provides package github.com/google/go-containerregistry/v1/random; to add it:
        go get github.com/google/go-containerregistry/v1/random
pkg/dgit/generate/generate.go:19:2: no required module provides package github.com/google/go-containerregistry/v1/remote; to add it:
        go get github.com/google/go-containerregistry/v1/remote
pkg/dgit/generate/generate.go:20:2: no required module provides package github.com/google/go-containerregistry/v1/remote/transport; to add it:
        go get github.com/google/go-containerregistry/v1/remote/transport
pkg/dgit/generate/generate.go:21:2: no required module provides package github.com/google/go-containerregistry/v1/types; to add it:
        go get github.com/google/go-containerregistry/v1/types
pkg/dgit/generate/generate.go:24:2: missing go.sum entry for module providing package golang.org/x/oauth2 (imported by matt-rickard.com/docker-merge/pkg/dgit/generate); to add:
        go get matt-rickard.com/docker-merge/pkg/dgit/generate
```

I installed go with homebrew but there still seems to be a bunch of packages missing. I've never used go so not sure how this might be fixed.

--- a/README.md
+++ b/README.md
@@ -65,7 +65,7 @@ docker build -t cowsay -f examples/getting-started/Dockerfile_cowsay .
 Now let's merge! We're going to tag the merged image as `cow-fortune` using the `-t` flag. The rest of the variadic arguments are images to merge, with the first being the image that "wins" in terms of merge conflicts.
 
 ```
-dgit -t cow-fortune fortune:latest cowsay:latest
+dgit merge -t cow-fortune fortune:latest cowsay:latest
 ```
 
 Finally, let's test it out!

